app/assets/javascripts/discourse/app/components/reviewable-field-text.js

```diff
@@ -0,0 +1,5 @@
+import Component from "@ember/component";
+
+export default Component.extend({
+  tagName: "",
+});
```

Comment:
We are generating new components in a few places like this, is it required now? I didn't see anything speak specifically to this in the code-mod.

Reply:
We may be able to change many of these to `templateOnly` components. When/if we colocate these templates we can probably entirely remove the backing class as well.